manifests/resource/upstream/member.pp

```diff
@@ -37,26 +37,11 @@
 define nginx::resource::upstream::member (
   $upstream,
   $server,
-  $ensure                 = 'present',
-  $port                   = 80,
+  Enum['present', 'absent'] $ensure = 'present',
```

Comment:
I would prefer it to directly drop it in the next release, since this has to be a major one we can get rid of a lot of stuff.